examples/example_transfer_transaction_ed25519.py

```diff
@@ -0,0 +1,29 @@
+from pactus.crypto import CryptoConfig
+from pactus.crypto.address import Address
+from pactus.crypto.ed25519.private_key import PrivateKey
+from pactus.transaction.transaction import Transaction
+from pactus.types.amount import Amount
+
+
+def main() -> None:
+    CryptoConfig.use_testnet()
+
+    lock_time = 1_735_096
+    memo = "This is a test transaction"
+    amount = Amount.from_string("1.5")
+    fee = Amount.from_string("0.01")
+    receiver = Address.from_string("tpc1ryz6m8meyfemyr4dhavz5aq067kkvs9mptpqnxr")
+
+    sender = Address.from_string("tpc1rv75w2y9hj64ht9spxx0e5s7avpclk9ey2eaavf")
+    sec = PrivateKey.from_string(
+        "tsecret1rglsgpylqrvet27azuvs9tsp8mpgf9lh4u4rkkarmcatfk9l0kucs7dcc09"
```

Comment:
Please make it capital. We try to keep Secret key capitals to highlight its security.

Reply:
Fixed.